app/assets/javascripts/student_profile/services_list.jsx

```diff
@@ -85,6 +85,7 @@ import _ from 'lodash';
 
       this.props.onClickDiscontinueService(serviceId);
       this.setState(this.getInitialState());
+      window.location.reload();
```

Comment:
So when I completely delete a service record after clicking on discontinue, the button would be stuck at updating. I notice that I have to refresh the page for that deleted item to disappear. I think the gif makes it feel like its not incredibly recognizable that the page actually refreshed which is good since you don't notice it.